src/models/m_rsc.erl

```diff
@@ -760,8 +760,9 @@ is_a(Id, Context) ->
     RscCatId = p(Id, category_id, Context),
     m_category:is_a(RscCatId, Context).
 
-%% @doc Return the categories and the inherited categories of the resource. Returns a list with category ids
--spec is_a_id(resource(), #context{}) -> list(atom()).
+%% @doc Return the categories and the inherited categories of the resource. Returns a list with
+%% category ids
+-spec is_a_id(resource(), #context{}) -> list(pos_integer()).
```

Comment:
The spec in line 765 uses a record, please define a type for the record and use that instead.